.github/workflows/ci.yml

```diff
@@ -15,23 +15,17 @@ jobs:
     strategy:
       matrix:
         react: [18, 19]
+      fail-fast: false
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: '22.x'
-          check-latest: true
-      - name: Set up git user
-        run: |
-          git config --global user.email 'action@github.com'
-          git config --global user.name 'GitHub Action'
-      - name: merge react 19 branch
+          node-version: '22.4.1'
+      - name: set up react 19
         if: matrix.react == 19
         run: |
-          git fetch origin react19
-          git merge origin/react19 -X theirs --allow-unrelated-histories
-          git fetch origin main
-          git diff origin/main
+          node ./.github/workflows/patch-react19.js
+          cat package.json
```

Comment:
Do you want to log the content?

Reply:
Keeping it in case we'll need it for debugging.